lib/src/metta/runner/stdlib.metta

```diff
@@ -193,3 +325,346 @@
+  (@params (
+    (@param "Anything")))
+  (@return "Unit atom"))
+
+(@doc let
+  (@desc "Let function is utilized to establish temporary variable bindings within an expression. It allows introducing variables (first argument), assign values to them (second argument), and then use these values within the scope of the let block")
+  (@params (
+    (@param "Variable name (or several variables inside brackets ())")
+    (@param "Expression to be bound to variable (it is being reduced before bind)")
+    (@param "Expression which will be reduced and in which variable (first argument) could be used")))
+  (@return "Result of third argument's evaluation"))
+
+(@doc let*
+  (@desc "Same as let, but first argument is a tuple containing tuples of variables and their bindings, e.g. (($v (+ 1 2)) ($v2 (* 5 6)))")
+  (@params (
+    (@param "Tuple of tuples with variables and their bindings")
+    (@param "Expression which will be reduced and in which variable (first argument) could be used")))
+  (@return "Result of second argument's evaluation"))
+
+(@doc new-state
+  (@desc "Creates a new state atom wrapping its argument")
+  (@params (
+    (@param "Atom to be wrapped")))
+  (@return "Returns (State $value) where $value is an argument to a new-state"))
+
+(@doc change-state!
+  (@desc "Changes input state's wrapped atom to another value (second argument). E.g. (change-state! (State 5) 6) -> (State 6)")
+  (@params (
+    (@param "State created by new-state function")
+    (@param "Atom which will replace wrapped atom in the input state")))
+  (@return "State with replaced wrapped atom"))
+
+(@doc get-state
+  (@desc "Gets a state as an argument and returns its wrapped atom. E.g. (get-state (State 5)) -> 5")
+  (@params (
+    (@param "State")))
+  (@return "Atom wrapped by state"))
+
+(@doc get-metatype
+  (@desc "Returns metatype of the input atom")
+  (@params (
+    (@param "Atom to get metatype for")))
+  (@return "Metatype of input atom"))
+
+(@doc register-module!
+  (@desc "Takes a file system path (first argument) and loads the module into the runner")
+  (@params (
+    (@param "File system path")))
+  (@return "Unit atom"))
+
+(@doc mod-space!
+  (@desc "Returns the space of the module (first argument) and tries to load the module if it is not loaded into the module system")
+  (@params (
+    (@param "Module name")))
+  (@return "Space name"))
+
+(@doc print-mods!
+  (@desc "Prints all modules with their correspondent spaces")
+  (@params ())
+  (@return "Unit atom"))
+
+(@doc sealed
+  (@desc "Replaces all occurrences of any var from var list (first argument) inside atom (second argument) by unique variable. Can be used to create a locally scoped variables")
+  (@params (
+    (@param "Variable list e.g. ($x $y)")
+    (@param "Atom which uses those variables")))
+  (@return "Second argument but with variables being replaced with unique variables"))
+
+; TODO: get-doc/help! not working for capture
```

Comment:
@DaddyWesker  could you please recheck and remove TODO from docs which work after merging https://github.com/trueagi-io/hyperon-experimental/pull/708

Reply:
@vsbogd 
That is exactly what I'm currently doing.